00-HTML-CSS-basics/HTML/Excersise1/tableOfExpenses.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
The table it's OK, but the requirement was to fill it with expenses. Anyway well done :)